docker/README.md

```diff
@@ -0,0 +1,61 @@
+# OSIM UI + Kerberos Tests Container 
+
+This is the container that is used to run the tests on the CI/CD pipeline. It is based on redhat's ubi9 image and has the necessary dependencies to run the tests.
+
+## Building the container
+Before building the container, you need to prepare some files.
+
+1. Create a `krb5.keytab` file in the `docker` directory. This file is used to authenticate with kerberos.
+```bash
+$ ktutil
+ktutil:  addent -password -p <principal> -k 1 -e aes256-cts-hmac-sha1-96 -f
+ktutil:  wkt krb5.keytab
+ktutil:  quit
+```
```

Comment:
```suggestion
`$ ktutil add --password='' -p <principal> -V 1 -e aes256-cts-hmac-sha1-96`
Is the macOs equivalent... there's no `-f` flag... what does that do in linux?
```

Reply:
the `-f` flag is for salt, it is also `-s`, which may be the one in MacOS. 

> Add  principal  to  keylist using key or password.
 If the -f flag is specified, salt information will be fetched from the KDC; in this case the -e flag may be omitted, or it may be supplied to force a particular enctype.
If the -f flag is not specified, the -e flag must be specified, and  the default salt will be used unless overridden with the -s option.

It might not be needed, I copied the command from OSIM readme https://github.com/RedHatProductSecurity/osim/tree/main/selenium#selenium-only-dockerpodman